js/scripts.js

```diff
@@ -0,0 +1,85 @@
+
+
+var state = {
+	"one": false,
+	"two": false,
+	"three": false,
+	"four": false,
+	"five": false,
+	"six": false,
+	"seven": false,
+	"eight": false,
+	"nine": false,
+};
+document.addEventListener("DOMContentLoaded", function(){
+	document.getElementById("one").addEventListener("click", buttonListener1);
+	document.getElementById("two").addEventListener("click", buttonListener1);
+	document.getElementById("three").addEventListener("click", buttonListener1);
+	document.getElementById("four").addEventListener("click", buttonListener1);
+	document.getElementById("five").addEventListener("click", buttonListener1);
+	document.getElementById("six").addEventListener("click", buttonListener1);
+	document.getElementById("seven").addEventListener("click", buttonListener1);
+	document.getElementById("eight").addEventListener("click", buttonListener1);
+	document.getElementById("nine").addEventListener("click", buttonListener1);
+	document.getElementById("clearB").addEventListener("click", clrListener);
+});
+
+var clrListener = function() {
+	event.preventDefault();
+	for (num in state)
+	{
+		state[num] = false;
+		document.getElementById(num).className = "sq";
+		document.getElementById(num).addEventListener("click", buttonListener1);
+		document.getElementById(num).innerHTML = "?";
+	}
+}
+
+var buttonListener1 = function (e) {
+	event.preventDefault();
+	console.log("worked");
+	console.log(e);
+	console.log(e.target.id);
+	if (e.target.innerHTML ==="X")
+	{
```

Comment:
Here, there's an issue with your logic. If a div contains 'X', we know that it has already been played. We don't want to change that value to 'O', but rather, alert the player that that space has already been played, and that they should click somewhere else.